mk/platform/Linux.mk

```diff
@@ -49,6 +49,24 @@ _OPSYS_EMULDIR.linux=	# empty
 _OPSYS_EMULDIR.linux32=	# empty
 
 # Support Debian/Ubuntu's multiarch hierarchy.
+
+###
+### Determine if multilib is available.
+###
+MULTILIB_SUPPORTED?=	unknown
+.if !empty(MACHINE_PLATFORM:MLinux-*-x86_64)
+.  if exists(/usr/include/x86_64-linux-gnu/gnu)
+_GNU_INCLUDE_DIR=	/usr/include/x86_64-linux-gnu/gnu
```

Comment:
You should add something like `_OPSYS_LIB_DIR`, on par with `_OPSYS_LIB_DIRS` and handle similarly.

Reply:
I have renamed to `_OPSYS_GNU_INCLUDE_DIR` on par with `_OPSYS_INCLUDE_DIRS`